src/components/number/number.jsx

```diff
@@ -55,12 +55,19 @@ function Number({
   const minimumFractionDigits = getFractionDigits(tickDecimals, valueMinDecimals, valueDecimals);
   const maximumFractionDigits = getFractionDigits(tickDecimals, valueMaxDecimals, valueDecimals);
   const formattedNumber = formatNumber(value, { minimumFractionDigits, maximumFractionDigits });
+  const absFormattedNumber = formatNumber(Math.abs(value), { minimumFractionDigits, maximumFractionDigits });
+  const ariaSign = (value < 0) ? 'minus ' : '';
```

Comment:
It did not appear to replace the "&minus;" with the minus sign on this place. So I changed it to just "minus" to make it "work" with voiceover.